common/wireguard/src/platform/linux/tun_device.rs

```diff
@@ -123,7 +123,7 @@ impl TunDevice {
 
         // This is how wireguard does it, by consulting the AllowedIPs table.
         if false {
-            let peers = self.peers_by_ip.lock().await;
+            let peers = self.peers_by_ip.as_ref().unwrap().lock().await;
```

Comment:
if you're just always unwrapping it, what's the point of wrapping it in an option in the first place?

Reply:
Well this is draft so ... ;)
Hold your horses

Reply:
🛑 🏇🏻

Reply:
There we go, no more unwrap :)